README.md

```diff
@@ -1148,19 +1148,39 @@ condition](#safe-assignment-in-condition).
 
   ```Ruby
   class Person
+    # bad
+    attr_reader(:name, :age)
+    # good
     attr_reader :name, :age
 
-    # omitted
+    # body omitted
   end
 
+  # bad
+  temperance = Person.new 'Temperance', 30
+  # good
   temperance = Person.new('Temperance', 30)
-  temperance.name
 
+  # bad
+  puts(temperance.age)
+  # good
   puts temperance.age
 
+  # bad
+  x = Math.sin y
+  # good
   x = Math.sin(y)
+
+  # bad
+  array.delete e
+  # good
   array.delete(e)
 
+  # bad
```

Comment:
I get the point of the final example, but this will generate a warning. That's one of the reasons `rspec` moved away from this "syntax". Using `eq` won't produce a warning.